server/services/statementService.js

```diff
@@ -9,28 +9,28 @@ module.exports = function createIncidentService({ incidentClient }) {
     return data.rows
   }
 
-  const getStatement = async (userId, incidentId, status) => {
-    const statement = await incidentClient.getStatement(userId, incidentId, status)
+  const getStatement = async (userId, reportId, status) => {
+    const statement = await incidentClient.getStatement(userId, reportId, status)
     if (!statement) {
-      throw new Error(`Incident: '${incidentId}' does not exist`)
+      throw new Error(`Report: '${reportId}' does not exist`)
     }
     return statement
   }
 
-  const validateSavedStatement = async (username, incidentId) => {
-    const statement = await getStatement(username, incidentId, StatementStatus.PENDING)
+  const validateSavedStatement = async (username, reportId) => {
+    const statement = await getStatement(username, reportId, StatementStatus.PENDING)
     const errors = statementConfig.validate ? validate(statementConfig.fields, statement, true) : []
     return errors
   }
 
-  const saveStatement = (userId, incidentId, statement) => {
-    logger.info(`Saving statement for user: ${userId} and incident: ${incidentId}`)
-    return incidentClient.saveStatement(userId, incidentId, statement)
+  const saveStatement = (userId, reportId, statement) => {
+    logger.info(`Saving statement for user: ${userId} and incident: ${reportId}`)
```

Comment:
Should this say 'and report'?

Reply:
these are both covered in the next PR...